### PR TITLE
set the height at small screen for the messages

### PR DESCRIPTION
**Pull request by shewood**
set the height to 60 for small screens to hopefully make it so you can see the message window.

### CODE REVIEW

File: src/components/organisms/MessagesContainer.vue [hunk 1]
```diff
@@ -1,12 +1,13 @@
 <template>
-  <div class="flex sm:h-vh-1/2">
+  <div class="flex md:h-screen">
```
Comment (nibivi77) [inline src/components/organisms/MessagesContainer.vue:2]: h-screen is too big in my opinion. Maybe 3/4 would be better?

File: src/components/organisms/MessagesContainer.vue [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
     <div
       class="
         w-full
         sm:w-2/5
         md:w-1/3
         sm:max-w-sm sm:border sm:border-r-0 sm:border-gray
         flex flex-col flex-shrink-0
+        px-1
```
Comment (nibivi77) [inline src/components/organisms/MessagesContainer.vue:10]: The px-1 does not reflect the Figma design so much and it creates a small white margin between the inbox box and the inbox items.
image

Comment (shewood) [reply src/components/organisms/MessagesContainer.vue:10]: I know,  but I wanted to see as on mobile it was getting clipped.

File: src/components/organisms/MessagesContainer.vue (hunk 1 above)
Comment (shewood) [reply src/components/organisms/MessagesContainer.vue:2]: maybe 3/4 would be better. what size do you find it too big?

Comment (nibivi77) [reply src/components/organisms/MessagesContainer.vue:2]: On mobile at 320px width, on landscape it looks like this:

**Review by nibivi77 — APPROVED** [review]
This looks good. Tested on multiple screen sizes.

**Pull request merged**